Documentation/BuildInstructionsLadybird.md

```diff
@@ -109,15 +109,11 @@ Qt chrome, install the Qt dependencies for your platform, and enable the Qt chro
 
 ```bash
 # From /path/to/ladybird
-cmake -S Meta/Lagom -B Build/lagom -DENABLE_QT=ON
+cmake -B Build/ladybird -DENABLE_QT=ON
```

Comment:
Not critical, but maybe we should reference the presets?
```
cmake --preset=default -DENABLE_QT=ON
```

Reply:
Ah good point. I think I'll do a follow-up with that as I update the IDE docs.